apps/shop/models.py

```diff
@@ -95,3 +96,41 @@ class ShopContactsModel(models.Model):
     email = models.CharField(max_length=100)
     viber_link = models.CharField(max_length=100)
     telegram_link = models.CharField(max_length=100)
+
+User = get_user_model()
+class Order(models.Model):
+    STATUS_CHOICES = [
+        ('new', 'Нове замовлення'),
+        ('confirmed', 'Підтверджено'),
+        ('shipped', 'Відправлено'),
+        ('delivered', 'Доставлено'),
+        ('complete', 'Виконано'),
+        ('canceled', 'Скасовано'),
+    ]
+    PAYMENT_CHOICES = [
+        ('online', 'Оплата онлайн'),
+        ('cod', 'Накладений платіж'),
+        ('postpay', ' После оплата')
+    ]
+
+    first_name = models.CharField(max_length=50)
+    last_name = models.CharField(max_length=50)
+    country = models.CharField(max_length=100)
+    region = models.CharField(max_length=100)
+    city = models.CharField(max_length=50)
+    delivery_service = models.CharField(max_length=100)
+    postoffice = models.CharField(max_length=50)
+    postbox = models.CharField(max_length=50)
+    payment_method = models.CharField(max_length=50, choices=PAYMENT_CHOICES)
+    phone_number = models.CharField(max_length=20)
+    email = models.EmailField()
+    comment = models.TextField(null=True, blank=True)
+    created_at = models.DateTimeField(auto_now_add=True)
+    update_at = models.DateTimeField(auto_now=True)
+    items = models.ManyToManyField('ItemModel', related_name='orders')
+    user = models.ForeignKey(User, related_name='orders', on_delete=models.SET_NULL, null=True, blank=True)
```

Comment:
We don't have regular users in system, only admin users, so no need in this field